.github/workflows/test-codecov.yaml

```diff
@@ -0,0 +1,27 @@
+name: Workflow for Codecov Testing
```

Comment:
Should this just be part of the existing CI workflow? as that is what generates the code coverage ?

Reply:
👋 Hey Aaron!

Yes, that would be perfect! This was just a test PR to check the `CODECOV_TOKEN` works. If you can point me to where the coverage data is generated I can do it myself (feel free to close this PR, not intended to be merged)